src/wp-includes/option.php

```diff
@@ -445,7 +445,6 @@ function update_option( $option, $value, $autoload = null ) {
 		$value = clone $value;
 	}
 
-	$value     = sanitize_option( $option, $value );
```

Comment:
By removing the `sanitize_option()` call here, the `pre_update_option{$option}` and `pre_update_option` hooks no longer pass the sanitized value to callbacks - a backward compatibility break.

Reply:
Gotcha. If I have another idea that would essentially revert this and take a new direction, is it best to close this pull request and open a new one, or can I just push a new commit here?

Reply:
Best to create a new PR so that each approach can be viewed historically (without needing to browse individual commits)

Reply:
Cool, thanks for the guidance.